sql/mob_spawn_mods.sql

```diff
@@ -1561,6 +1561,12 @@ INSERT INTO `mob_spawn_mods` VALUES (17092904,1,1800,1); -- min gil 1800
 INSERT INTO `mob_spawn_mods` VALUES (17092904,2,5800,1); -- max gil 5800
 INSERT INTO `mob_spawn_mods` VALUES (17092904,15,2500,1); -- mug gil 2500
 
+-- Peg Powler
+-- Based on wiki (https://ffxiclopedia.fandom.com/wiki/Peg_Powler)
+INSERT INTO `mob_spawn_mods` VALUES (17649761,1,6000,1); -- min gil 6000
+INSERT INTO `mob_spawn_mods` VALUES (17649761,2,9200,1); -- max gil 9200
+INSERT INTO `mob_spawn_mods` VALUES (17649761,15,4500,1); -- mug gil 4500
```

Comment:
For any named NM I would much rather have these inside the Lua script onspawn. I am a big fan of getting mods out of sql for NMs so we do not have to look at 5 different sqls to find out it has a secret mod somewhere. You can see any of the HNM scripts as an example.